### PR TITLE
Update sponsors list

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: sponsors.md [hunk 1]
```diff
@@ -79,3 +77,17 @@ Thank you to Enrico, who provided ARM and ARM64 could servers for our package bu
 
 <div class="lead text-left"><a href="http://www.heystephenwood.com">Hey Stephen Wood</a></div>
 Thank you to Stephen, who provided a server for our continuous integration system.
+
+### Rackspace
+
+<div class="lead text-left"><a href="https://www.rackspace.com"><img src="/static/images/sponsors/rackspace.jpg" /></a></div>
+Thank you to Rackspace, who provide infrastructure on Rackspace Cloud to power our continuous integration system and automated testing.
```
Comment (tbrisker) [inline sponsors.md]:
```suggestion
Thank you to Rackspace, who provided infrastructure on Rackspace Cloud to power our continuous integration system and automated testing.
```

File: sponsors.md [hunk 2]
```diff
@@ -79,3 +77,17 @@ Thank you to Enrico, who provided ARM and ARM64 could servers for our package bu
...
+
+### oVirt
+
+<div class="lead text-left"><a href="http://www.ovirt.org"><img src="/static/images/sponsors/oVirt.svg" height="100px"/></a></div>
+Thank you to oVirt, who provide servers for our continuous integration system.
```
Comment (tbrisker) [inline sponsors.md]:
```suggestion
Thank you to oVirt, who provided servers for our continuous integration system.
```

File: sponsors.md [hunk 3]
```diff
@@ -79,3 +77,17 @@ Thank you to Enrico, who provided ARM and ARM64 could servers for our package bu
...
+
+### Scaleway
+<div class="lead text-left"><a href="https://www.scaleway.com/"><img src="/static/images/sponsors/scaleway.svg" height="100px"/></a></div>
+Thank you to Scaleway, who provide ARM servers for building our arm packages.
```
Comment (tbrisker) [inline sponsors.md]:
```suggestion
Thank you to Scaleway, who provided ARM servers for building our arm packages.
```

File: sponsors.md (hunk 1 above)
Comment (ekohl) [inline sponsors.md]: Other former sponsors are in the past tense
```suggestion
Thank you to Rackspace, who provided infrastructure on Rackspace Cloud to power our continuous integration system and automated testing.
```

Same below

**Review by tbrisker — CHANGES_REQUESTED** [review]
past sponsors should be past tense

**Review by ekohl — CHANGES_REQUESTED** [review]

File: sponsors.md (hunk 3 above)
Comment (evgeni) [reply sponsors.md]: Technically, we also run Discourse and Redmine there.

File: sponsors.md [hunk 4]
```diff
@@ -49,6 +35,18 @@ Thank you to dm, for their many ongoing contributions to the codebase.
...
+<div class="lead text-left"><a href="https://osuosl.org"><img src="/static/images/sponsors/osuosl.png" height="125px"/></a></div>
+Thank you to OSUOSL, who provides infrastructure to power our continuous integration system and automated testing.
+
+### CentOS CI
+<div class="lead text-left"><a href="https://wiki.centos.org/QaWiki/CI"><img src="/static/images/sponsors/centos.png" height="125px"/></a></div>
+Thank you to CentOS CI, who provides Jenkins infrastructure and bare metal hardware for running integration tests for release pipelines.
+
+### gandi.net
+<div class="lead text-left"><a href="https://gandi.net"><img src="/static/images/sponsors/gandi.jpg" height="125px"/></a></div>
+Thank you to gandi.net, who provides the DNS that powers all of our websites and infrastructure.
```
Comment (evgeni) [inline sponsors.md]:
```suggestion
Thank you to gandi.net, who provides the domain registration and DNS that powers all of our websites and infrastructure.
```

File: sponsors.md [hunk 5]
```diff
@@ -49,6 +35,18 @@ Thank you to dm, for their many ongoing contributions to the codebase.
 <div class="lead text-left"><a href="https://fastly.com"><img src="/static/images/sponsors/fastly.png" height="125px"/></a></div>
 Thank you to Fastly, who provides the CDN for our downloads and repositories.
 
+### OSUOSL
+<div class="lead text-left"><a href="https://osuosl.org"><img src="/static/images/sponsors/osuosl.png" height="125px"/></a></div>
+Thank you to OSUOSL, who provides infrastructure to power our continuous integration system and automated testing.
+
+### CentOS CI
```
Comment (evgeni) [inline sponsors.md]: I wonder if we should thank CentOS CI or CentOS or Fedora Infra.

Comment (ehelms) [reply sponsors.md]: I debated just CentOS since that is the wiki/website/name/mailing list.

File: sponsors.md [hunk 6]
```diff
@@ -79,3 +77,17 @@ Thank you to Enrico, who provided ARM and ARM64 could servers for our package bu
...
+Thank you to Rackspace, who provided infrastructure on Rackspace Cloud to power our continuous integration system and automated testing.
+
+### oVirt
+
+<div class="lead text-left"><a href="http://www.ovirt.org"><img src="/static/images/sponsors/oVirt.svg" height="100px"/></a></div>
+Thank you to oVirt, who provided servers for our continuous integration system.
+
+### Scaleway
+<div class="lead text-left"><a href="https://www.scaleway.com/"><img src="/static/images/sponsors/scaleway.svg" height="100px"/></a></div>
+Thank you to Scaleway, who provided ARM servers for building our arm packages.
```
Comment (tbrisker) [inline sponsors.md]:
```suggestion
Thank you to Scaleway, who provided ARM servers for building our ARM packages.
```

**Comment by tbrisker** [comment]
Thanks @ehelms, fixed the ARM capitalization for you :)

**Pull request merged**